drivers/hw_cc310/Kconfig

```diff
@@ -21,6 +21,7 @@ if !HW_CC3XX_FORCE_ALT
 
 menuconfig HW_CC3XX
 	bool "Arm CC3xx hw driver for Nordic devices"
+	depends on !TRUSTED_EXECUTION_NONSECURE
 	depends on SOC_NRF52840 || (SOC_NRF9160 && !SPM) || (SOC_NRF9160 && TRUSTED_EXECUTION_SECURE) || \
```

Comment:
Can this be simplified to: **depends on SOC_NRF52840 || !TRUSTED_EXECUTION_NONSECURE**

Reply:
depends on !TRUSTED_EXECUTION_NONSECURE && (HAS_HW_NRF_CC310 || HAS_HW_NRF_CC312)